websockets/framing.py

```diff
@@ -14,10 +14,18 @@
 import io
 import random
 import struct
+from itertools import cycle
 
 from .exceptions import PayloadTooBig, WebSocketProtocolError
 
 
+try:
+    from .cext import apply_mask
+except ImportError:
+    def apply_mask(mask, data):
+        return bytes(b ^ m for b, m in zip(data, cycle(mask)))
```

Comment:
BTW, great work optimizing the Python version! I find it more readable and it's faster as well!

```python
import timeit

timeit.timeit(
    stmt='bytes(b ^ mask[i % 4] for i, b in enumerate(data))',
    setup='import os; data = os.urandom(4000); mask = os.urandom(4);',
    number=1000,
)

# 0.66

timeit.timeit(
    stmt='bytes(b ^ m for b, m in zip(data, itertools.cycle(mask)))',
    setup='import itertools; import os; data = os.urandom(4000); mask = os.urandom(4);',
    number=1000,
)

# 0.44
```